HIRS_Utils/src/main/java/hirs/data/persist/certificate/Certificate.java

```diff
@@ -365,7 +365,8 @@ public Certificate(final byte[] certificateBytes) throws IOException {
                         .getInstance((DLSequence) getExtensionValue(
                                 Extension.authorityKeyIdentifier.getId()));
 
-                this.authorityInfoAccess = getAuthorityInfoAccess();
+                this.authorityInfoAccess = getAuthorityInfoAccess(x509Certificate
```

Comment:
Would it be possible to get some test coverage of at least one of these two `switch` paths (likely the X509 Certificate) and assert that this is being parsed and stored correctly?

Reply:
There are parsing unit tests for both switch statements.  Are you asking for a line that checks the authority info access specifically?

Reply:
Yeah that would be good.